zebra-consensus/src/primitives/groth16/params.rs

```diff
@@ -101,6 +101,8 @@ impl Groth16Parameters {
                 .expect("unreachable code: sprout loader panics on failure"),
         };
 
+        tracing::info!("Zcash Sapling and Sprout parameters downloaded and/or verified");
```

Comment:
Optional: We already log when the parameters are downloaded in `zebra-consensus`. Should we delete the `zebra-consensus` log as part of this change?

`zebra-consensus` only logs when it is run by `zebrad start`, so I think this log is better.

https://github.com/ZcashFoundation/zebra/blob/f3f8254e4ad84c14fb3f0cbd4386df4ef4795fb0/zebra-consensus/src/chain.rs#L248

Reply:
https://github.com/ZcashFoundation/zebra/pull/5901/commits/db847d67adf569a4374a4bfa740d9fddc5ab9154